### PR TITLE
Update store app description

### LINKED ISSUE #437: Fill in description in fastlane metadata structure
Right now it just has the tagline which probably isn't enough.

https://github.com/evilcorpltd/aTox/blob/master/fastlane/metadata/android/en-US/full_description.txt
https://github.com/evilcorpltd/aTox/blob/master/fastlane/metadata/android/en-US/short_description.txt

This is the blocking publishing on Google Play and FDroid.

### PR DESCRIPTION
**Pull request by robinlinden**
Resolves #437

### CODE REVIEW

**Review by JFreegman — APPROVED** [review]

File: fastlane/metadata/android/en-US/full_description.txt [hunk 1]
```diff
@@ -1 +1,11 @@
-The reasonable Tox client for Android.
+<p><b>aTox, a beautiful and modern Tox client.</b></p>
+
+<p>aTox is a free open-source messaging app for the Tox protocol.</p>
+
+<p><b>Why aTox?</b></p>
+<ul>
+<li>Open-source: free to share, inspect, and change as you like.</li>
```
Comment (GrayHatter) [inline fastlane/metadata/android/en-US/full_description.txt:7]:
```
 <li>Open-source: free to share, inspect, and change as you like.</li>
 <li>Reasonable: aTox uses sane defaults, doesn't make guess about what you care about, and doesn't ignore what <b>you</b> want</li>
```

**Review by GrayHatter — CHANGES_REQUESTED** [review]

File: fastlane/metadata/android/en-US/full_description.txt (hunk 1 above)
Comment (robinlinden) [reply fastlane/metadata/android/en-US/full_description.txt:7]: Added, but reworded to fit under 120 cols. Thank you!

File: fastlane/metadata/android/en-US/full_description.txt [hunk 2]
```diff
@@ -1 +1,12 @@
...
+<li>Reasonable: sane defaults, no guesses about what you care about, doesn't ignore what <b>you</b> want.</li>
+<li>End-to-end encrypted: the only people who can see your conversations are the people you're talking to.</li>
+<li>Distributed: no central servers that can be taken down or forced to turn over data.</li>
```
Comment (GrayHatter) [inline fastlane/metadata/android/en-US/full_description.txt:10]: s/forced/compelled

**Review by GrayHatter — APPROVED** [review]

File: fastlane/metadata/android/en-US/full_description.txt (hunk 2 above)
Comment (robinlinden) [reply fastlane/metadata/android/en-US/full_description.txt]: Fixed. Also added links both to the aTox GitHub and https://tox.chat for more info about the app and the protocol like you suggested on IRC.

**Pull request merged**